libs/sdk-backend-tiger/src/backend/workspaces/index.ts

```diff
@@ -52,8 +53,9 @@ class TigerWorkspaceQuery implements IWorkspacesQuery {
         return this;
     }
 
-    public withParent(workspaceId: string): IWorkspacesQuery {
-        this.parentWorkspaceId = workspaceId;
+    public withParent(workspaceId: string | undefined): IWorkspacesQuery {
+        // isnull=true - get only root workspaces
+        this.parentWorkspaceId = workspaceId ? workspaceId : "isnull=true";
```

Comment:
I think the app can pass `isnull=true` when it wants to fetch root workspaces only.  For a non-admin user, the app simply passes undefined as `workspaceId` to fetch all workspaces (root and non-root).

→ Please revert the changes here.